store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedAccess.java

```diff
@@ -72,13 +73,13 @@
  */
 @JsonDeserialize(builder = FederatedAccess.Builder.class)
 @JsonPropertyOrder(value = {"class", "owningUserId"}, alphabetic = true)
-public class FederatedAccess implements AccessControlledResource, Serializable {
```

Comment:
As I understand it, this does not need to be Serializable anymore because of the transient stuff? Just checking this will still work for JCS Cache

Reply:
It is JSON serialisable.

but the transient stuff turns the AccessPredicte to string just before it pops in the cache (JCS or what ever) and visa versa